.github/workflows/build-container.yml

```
@@ -0,0 +1,33 @@
+name: dispatched build-docker-image
+
+on:
+  repository_dispatch:
+    types:
+      - dispatch-build
+
+jobs:
+  dispatched-build:
```

```suggestion
  build:
```

Since we want to build this on dispatch + push